packages/suite-desktop-core/e2e/support/common.ts

```diff
@@ -125,3 +125,6 @@ export const calculatePercentageOfBalance = (params: PercentageOfBalanceParams)
 
     return localizeNumber(fraction, 'en', 0, maxDecimals);
 };
+
+export const createWaitPromise = (timeout: number = 1000) =>
```

Comment:
I'd be glad for an advice, coz without it , last time i Checked, the test are randomly failing even though they see the element they should click.